hedera-node/hedera-consensus-service-impl/src/main/java/com/hedera/node/app/service/consensus/impl/codecs/ConsensusServiceStateTranslator.java

```diff
@@ -101,7 +107,9 @@ public static com.hedera.node.app.service.mono.state.merkle.MerkleTopic pbjToSta
                                 .orElse(null),
                         topic.autoRenewPeriod(),
                         new com.hedera.node.app.service.mono.state.submerkle.EntityId(
-                                0, 0, topic.autoRenewAccountNumber()),
+                                topic.autoRenewAccountId().shardNum(),
+                                topic.autoRenewAccountId().realmNum(),
+                                topic.autoRenewAccountId().accountNum()),
```

Comment:
Similarly, we can use EntityId.fromGrpcAccountId(topic.autoRenewAccountId()) here

Reply:
EntityId.fromGrpcAccountId() accepts com.hederahashgraph.api.proto.java.AccountID, topic.autoRenewAccountId() is com.hedera.hapi.node.base.AccountID. Kept as it is.

Reply:
Then I would suggest adding `fromPbjAccountId` to `EntityId` for future reuse instead of duplicating code.

Reply:
Good suggestion. Updated. Thanks.